### ISSUE TITLE
PostSonarr.py not adding to Sonarr

### ISSUE THREAD

**Issue by T0rm3nt** [mdhiggins/sickbeard_mp4_automator#1118]
Hi,

After searching around on here I've found many fixes & suggestions for my error however none of them have worked so far!

Im using Sonarr V3 & python 2.7 on windows.
On V3 they have now removed the arguments field from the connect settings so I have tried to add the script directly as suggested elsewhere. Every time i try to add it i receive the following error;
"Script exited with code: 1"
Ive double checked and the PostSonarr.py script I am using does have the sys.exit(0) at the end.
The problem with it is that it will not let me save or add the script until it has ran the 'test' phase.
Any ideas would be greatly appreciated as this has been driving me crazy!

![Annotation 2020-01-28 151933](https://user-images.githubusercontent.com/60298506/73277127-9adf6480-41e1-11ea-8ac4-448b56162ba7.png)

**Comment by mdhiggins** [comment]
Do the Sonarr logs give any indication if there's python errors occurring?

The 'test' scenario has been accounted for in the latest version of the script for a while now
```
if os.environ.get('sonarr_eventtype') == "Test":
    sys.exit(0)
```
So its not that there's no direct support of the test and appropriate exit code. If there's a python error, like a dependency missing that will cause error code 1 before the test can be fired. My thought is that with this new change it may be using the wrong python environment or failing to execute the python file all together

2 possible suggestions here (though I admitted don't have a sonarr v3 test environment set up yet so you'll have to test for me and report back)

1. Try including the python exe path, essentially merging the path and argument parameters that would have previously been separated. For example `C:\Python\Python.exe C:\mp4Automator\postSonarr.py`
2. If the above fails and Sonarr doesn't like that, make a .bat file that includes that exact path above and see if that cooperates

Let me know

**Comment by T0rm3nt** [comment]
@mdhiggins thank you for the prompt and detailed reply 👍 
This was of my own doing and tinkering with stuff i shouldn't have so appreciate you trying to help!
You were absolutely right even though the script was passing the exit 0 code as it was meant to it was clearly hitting another error along the way which then changed it to give a 1 code. 
This of course was going unnoticed on my V2 setup as i wasn't checking the logs regularly however once id moved to V3 and had to re-do my connect settings it then made the problem apparent as it wouldn't let me save the new config.

Long story short I foolishly didn't notice that the tmdbsimple dependency was throwing an error in the Sonarr logs, so after quickly re-installing this dependency it of course worked as expected and let me save the settings by putting the path directly to the script.

Thanks again for your help and also for this awesome script !
